griptape/mixins/futures_executor_mixin.py

```diff
@@ -1,14 +1,13 @@
 from __future__ import annotations
 
-from abc import ABC
 from concurrent import futures
 from typing import Callable, Optional
 
 from attrs import Factory, define, field
 
 
 @define(slots=False, kw_only=True)
-class FuturesExecutorMixin(ABC):
+class FuturesExecutorMixin:
```

Comment:
Did you remove `ABC` for "style" (like because this class has no abstract methods) or for function (because you got some error)?